packages/ember-cli-code-coverage/index.js

```diff
@@ -48,19 +58,25 @@ module.exports = {
       if (config.coverageEnvVar) {
         coverageEnvVar = config.coverageEnvVar;
       }
+
+      if (config.extension) {
```

Comment:
@vstefanovic97 what is the use case when we want to configure this?
Also, as this exposes new option for consumers, let's document it in readme

Reply:
@SergeAstapov I can't think of a use case right now, but thought maybe it would be useful to expose it still, I'll add it to the docs

Reply:
@SergeAstapov I added this option to the docs